### DISCUSSION TITLE
The model is gaining water (WaterBalanceError is positive)

### DISCUSSION THREAD

**Discussion by liu-13** [NCAR/hrldas#205] [General]
When I change the soil moisture in the restart file, it reports an error. But the modified soil moisture is only 0.0001 from the original. It works fine without modifying the RESTART file.
/
![QQ图片20240815104326](https://github.com/user-attachments/assets/eb3e7511-539c-401f-bf4a-9ed73a47ecf0)

**Comment by cenlinhe** [comment]
which variable did you change for soil moisture? Did you change the total soil moisture and soil liquid water together?

**Reply by liu-13** [reply]
What I changed was the SMC variable in the RESTART file

**Reply by liu-13** [reply]
Another problem is that the pattern works fine if only the SMC variable in the RESTART file is +0.01 only, and all the grid points are +0.01. But according to my processing it doesn't work, and the strangest thing is that the amount I changed is not even 0.01 large enough

**Reply by cenlinhe** [reply]
It looks like something might be wrong with your scripts adding soil moisture to the restart file? because your updated restart file after adding SMC is not equal to the amount you want to add.
Also, please add the same amount to SMC and SH2O so that soil ice content will be unchanged.
Once you update your restart file after running your scripts of adding soil moisture, please double check if the file includes the correct SMC and SH2O values as you expected before using it to run your model.
Another thing to note is that you cannot change the soil moisture to exceed the saturated soil moisture (a key soil property in NoahmpTable.TBL).

**Reply by liu-13** [reply]
Yes, you are right. This is no longer an issue after the liquid water content modification. Thank you.
